tests/ha/multi_pod_failure/test_multi_data_pod_failure.py

```diff
@@ -896,107 +896,127 @@ def test_chunk_upload_during_data_kpods_down(self):
         LOGGER.info("ENDED: Test chunk upload during k data pods going down by delete "
                     "deployment (unsafe)")
 
-    # pylint: disable=too-many-locals
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.tags("TEST-35777")
     def test_reads_writes_during_kpods_down(self):
         """
         This test tests continuous READs/WRITEs while pods are failing till K data pods are failed
         """
-        LOGGER.info("STARTED: Test to verify continuous READs/WRITEs while %s (K) pods "
+        LOGGER.info("STARTED: Test to verify continuous READs/WRITEs while %s (K) data pods "
                     "were going down.", self.kvalue)
 
-        LOGGER.info("Step 1: Start continuous IOs with variable object sizes in background")
+        event = threading.Event()  # Event to be used to send intimation of data pod deletion
         users = self.mgnt_ops.create_account_users(nusers=1)
-        self.test_prefix = 'test-35777'
         self.s3_clean = users
-        output = Queue()
-        event = threading.Event()
-        args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
-                'nclients': 2, 'nsamples': 30, 'skipcleanup': True, 'output': output}
 
-        thread = threading.Thread(target=self.ha_obj.event_s3_operation,
-                                  args=(event,), kwargs=args)
-        thread.daemon = True  # Daemonize thread
-        thread.start()
-        time.sleep(HA_CFG["common_params"]["30sec_delay"])
-        LOGGER.info("Step 1: Successfully started continuous IOs with variable object sizes in "
+        LOGGER.info("Step 1: Perform READs/WRITEs with variable object sizes during "
+                    "data pod down by delete deployment.")
+        LOGGER.info("Step 1.1: Perform WRITEs with variable object sizes for parallel READs")
+        test_prefix_read = 'test-read-35777'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=test_prefix_read, skipread=True,
```

Comment:
Here also, we should do read-verify as well.

Reply:
> Here also, we should do read-verify as well.

same as #1745 ,